Add InSpec 1.0.0

jvrplmlmn opened the pull request:
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

vitorgalvao commented on Casks/inspec.rb:
```diff
@@ -0,0 +1,25 @@
+  url "https://packages.chef.io/stable/mac_os_x/10.11/inspec-#{version}.dmg"
+  appcast 'https://github.com/chef/inspec/releases.atom',
+          checkpoint: 'c9905934b7e515e3f0486187dabee2bb43dc7a63677f0fdc0ca85bc0f316597a'
+  name 'InSpec'
```
Remove this one.

vitorgalvao commented on Casks/inspec.rb:
```diff
@@ -0,0 +1,25 @@
+                       '/opt/inspec/',
+                       '/usr/local/bin/inspec',
+                     ]
+  zap delete: '~/.inspec/'
```
This did not pass `style`. It needs an extra line above.

vitorgalvao commented on Casks/inspec.rb:
```diff
@@ -0,0 +1,25 @@
+
+  pkg "inspec-#{version}.pkg"
+
+  # When updating this cask, please verify the list of paths to delete (and
```
Remove these comments and instead add an [`uninstall_postflight`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/flight.md) ([example](https://github.com/caskroom/homebrew-cask/blob/f2fcaa324c3a1f8bd9eab9261963b3ffdad10c65/Casks/virtualbox-extension-pack.rb#L22L24).

In it, give the comment “As suggested in https://docs.chef.io/install_dk.html#mac-os-x”.

jvrplmlmn commented:
Thanks for your quick response @vitorgalvao 

I've addressed your comments and fix the style issue. Still waiting for the CI to finish.

Let me know if you want me to squash all the commits if you consider this ready to merge.

vitorgalvao commented:
Thank you for the quick changes.

No need to squash, as github now allows us to do that on merge. In fact, *not* squashing is now preferred, since it makes it easier to see the changes from the last time we looked at it.
